infra/recipes/docker-compose/common/spark/spark-base-hadoop3.2.dockerfile

```diff
@@ -81,7 +81,7 @@ RUN mkdir -p "${LIVY_HOME}/logs"
 
 COPY /infra/recipes/docker-compose/common/spark/start-spark.sh /
 COPY /build/openhouse-spark-runtime_2.12/libs/*[^sources][^javadoc].jar $SPARK_HOME/openhouse-spark-runtime_2.12-latest-all.jar
-COPY /build/openhouse-spark-apps_2.12/libs/openhouse-spark-apps_2.12-all.jar $SPARK_HOME/openhouse-spark-apps_2.12-latest-all.jar
+COPY /build/openhouse-spark-apps_2.12/libs/openhouse-spark-apps_2.12-uber.jar $SPARK_HOME/openhouse-spark-apps_2.12-latest-all.jar
```

Comment:
Are we changing the default "all" classifier to "uber" for fat jar? If so changing classifier won't work with gradle 7. Please share the previous PR where this change is done. We might need to check with gradle team on this.

Reply:
https://github.com/linkedin/openhouse/pull/91 ! oh no~ we just now were able to unblock after a 2-3week effort 🤣